dev/ts/generate

```diff
@@ -7,4 +7,4 @@ set -e
 )
 
 rm -f ./ts/**/*.pb.ts
-docker run --rm -i -v${PWD}:/code xmtp/protoc -I=./proto --grpc-gateway-ts_out=ts message_api/v1/message_api.proto
\ No newline at end of file
+docker run --rm -i -v${PWD}:/code xmtp/protoc -I=./proto --grpc-gateway-ts_out=ts message_api/v1/message_api.proto message_api/v1/authn.proto
```

Comment:
We are going to want to change this to use `protoc-gen-ts_proto` for the `authn` protos to get real protobuf generated code. This is going to generate types and code for a GRPC Gateway JSON API.

Think we'll need to add the `--ts_proto_opt=esModuleInterop=true` flag as well

Reply:
Can you suggest the change you have in mind. This stuff is black magic to me.

Reply:
I'll try and commit it to this branch. It'll take a little fiddling for me to figure out too. Protoc arguments never work the way I expect them to.

Reply:
You can see the changes I made [here](https://github.com/xmtp/proto/pull/7/commits/4a3ff12b35442c2de0b89ad19bd2cb6a30a69f0e)

Basically, I added the generator to the Docker image and run it as a second codegen step, since the output formats are different. 

This build is getting complicated enough I am starting to wonder if using `buf` would make this more maintainable/readable. I hesitated to include it originally, since it's giving us very little value in `xmtp-js`. But maybe it makes sense here just to make this configuration more declarative.

Reply:
Thank you! I'd rather postpone switching to buf for a separate PR.

Reply:
Makes sense. I can take that one.